content/schemas/sign_request_prefill_tag.yml

```diff
@@ -0,0 +1,28 @@
+---
+title: Sign Request Prefill Tag
+
+type: object
+
+description: |-
+  Prefill tags used to prefill signer input data
+
+properties:
+  external_id:
+    type: string
+    example: '1234'
+    description: This references the id of a particular tag added to the 
```

Comment:
I'm a bit confused how the client can get this ID, since they are expected to send it prefill_tags in creating a sign request.

Reply:
This is something that the client sets in the body of the file before uploading it